tests/analysis/polarization/test_polarization.py

```diff
@@ -36,14 +36,14 @@ def test_polarization(enable_backend, full_graph_edgelist: pd.DataFrame):
     assert_frame_equal(scores, expected)
 
 
-# TODO Add test here that tests networkx backend
 def test_polarization_with_differentially_polarized_markers():
     # Set seed to get same graph every time
     graph = create_randomly_connected_bipartite_graph(
-        n1=50, n2=100, p=0.1, random_seed=1477
+        n1=50, n2=100, p=0.1, random_seed=2
```

Comment:
WHy the difference in seed here? Is it important?

Reply:
Using 1477 as a random seed with networkx here created a graph with two components, so I changed the random seed until I found one that generated a single connected component with the same settings for n1, n2, and p.